src/apiCalls.js

```diff
@@ -0,0 +1,9 @@
+export const getResults = (coords) => {
+  return fetch('https://fe-cors-proxy.herokuapp.com', {
+  headers: {
+    "Target-URL": `https://boneyard-be.herokuapp.com/api/coordinates=${coords.latitude},${coords.longitude}`
+  }
+})
```

Comment:
Let's coordinate with BE to ensure that we can access the target endpoint without having to use a proxy.